### ISSUE TITLE
fix cron script

### ISSUE THREAD

**Issue by robkooper** [PecanProject/pecan#1515]
Make sure the cron script is generic enough and/or accepts parameters to be used. This right now works on the VM but not on other machines (see https://github.com/PecanProject/pecan/pull/1494#discussion-diff-126249885R87).

**Comment by infotroph** [comment]
@robkooper safe to close now that DB sync is rare?

**Comment by robkooper** [comment]
yeah lets close. I'd almost say we remove the dbsync shiny app as well
